### PR TITLE
Update periphery in UR

### PR DESCRIPTION
**Pull request by hensha256**

### CODE REVIEW

File: contracts/modules/Payments.sol [hunk 1]
```diff
@@ -41,14 +40,13 @@ abstract contract Payments is PaymentsImmutables {
     /// @param recipient The address that will receive payment
     /// @param bips Portion in bips of whole balance of the contract
     function payPortion(address token, address recipient, uint256 bips) internal {
-        if (bips == 0 || bips > FEE_BIPS_BASE) revert InvalidBips();
```
Comment (snreynolds) [inline contracts/modules/Payments.sol:44]: Do you not want to check bips == 0 ?

Comment (hensha256) [reply contracts/modules/Payments.sol:44]: I dont really see that it gains anything

**Review by snreynolds — COMMENTED** [review]
🕺 🕺 🕺

**Review by snreynolds — APPROVED** [review]
lgtm

**Pull request merged**